source/appModules/securecrt.py

```diff
@@ -16,7 +15,10 @@
 class AppModule(appModuleHandler.AppModule):
 
 	def chooseNVDAObjectOverlayClasses(self, obj, clsList):
-		if re.match("AfxFrameOrView\d{2,}u", obj.windowClassName) and obj.IAccessibleRole == oleacc.ROLE_SYSTEM_CLIENT:
+		#initialize a compiles regexp object for all window names following the standard SecureCrt Window naming comvention for all versions
+		MATCHER = re.compile ("AfxFrameOrView\d{2,}u")

```

Comment:
Because this is inside the `__init__` method, it's still going to compile every time the method runs. Instead, put it at module level (above the class). This way, it will only compile when the module is imported.